src/filters/madgwickahrs.cpp

```diff
@@ -51,9 +51,7 @@ namespace dynamicgraph
       : Entity(name)
       ,CONSTRUCT_SIGNAL_IN( accelerometer,            dynamicgraph::Vector)
       ,CONSTRUCT_SIGNAL_IN( gyroscope,                dynamicgraph::Vector)
-      ,CONSTRUCT_SIGNAL_OUT(imu_quat,                 dynamicgraph::Vector,
-			    m_gyroscopeSIN <<
-			    m_accelerometerSIN)
+      ,CONSTRUCT_SIGNAL_OUT(imu_quat,                 dynamicgraph::Vector, m_gyroscopeSIN << m_accelerometerSIN)
```

Comment:
Same comment

Reply:
On the naming convention? As you can see, this signal was already there, I did not choose the name.

Reply:
Oh, you mean the line length! Ok